### DISCUSSION TITLE
Manipula imágenes en Blogger fácilmente

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#23] [Announcements]
# Manipula imágenes en Blogger fácilmente

Hoy te enseñare a crear una función para manipular las imágenes alojadas en Google a traves de Blogger, Google Fotos e incluso Youtube

https://www.zkreations.com/2023/09/manipula-imagenes-en-blogger-facilmente.html

**Comment by eltorga** [comment]
amigo, tienes talvez las reacciones personalizadas de sharethis, me gustaria usarlas

**Reply by danieIabel** [reply]
que tal amigo, ya no tengo esa publicación, tampoco fomento ya el uso de sharethis para ese motivo, ya que recopila muchos datos y no son realmente útiles las reacciones ya que no posicionan los posts ni tampoco se pueden usar esos datos para crear algo con ellos. Trabajaré en una alternativa que no dependa de terceros.

**Comment by danieIabel** [comment]
He actualizado el artículo agregando soporte para el servidor proxy de Google, el cual es el que se usa cuando la imagen no está subida a Blogger o se trata de un video de youtube. Gracias a este cambio en la función, se le pueden aplicar parámetros a estos enlaces.

**Comment by danieIabel** [comment]
He actualizado nuevamente el funcionamiento de esta inclusion, ahora se detecta los servidores que permiten parámetros de manera mas eficiente.
